cmd/soroban-cli/src/commands/contract/read.rs

```diff
@@ -87,7 +88,7 @@ pub enum Error {
     NoContractDataEntryFoundForContractID,
     #[error(transparent)]
     Key(#[from] key::Error),
-    #[error("Only contract data and code keys are allowed")]
+    #[error("Only contract code, data, and code keys are allowed")]
```

Comment:
It already says "code".
```suggestion
    #[error("Only contract data, and code keys are allowed")]
```